src/client/components/MapHeader.tsx

```diff
@@ -247,6 +256,7 @@ const MapHeader = ({
                     sx={{ ...style.selectionButton }}
                     className={buttonClassName(selectionTool === tool)}
                     onClick={() => {
+                      // Open slider on click if not already open, close it otherwise
                       setPaintBrushSizeSliderVisibility(
                         tool === SelectionTool.PaintBrush && !isPaintBrushSizeSliderVisible
                       );
```

Comment:
I don't think we want this behavior - we want the menu to be open so long as you have the paint brush tool selected, right?
```suggestion
                      // Open slider on click if not already open
                      setPaintBrushSizeSliderVisibility(tool === SelectionTool.PaintBrush);
```

Reply:
Hmm, I personally like the ability to toggle the paintbrush slider visibility by clicking on the icon - when you're not using it it's nice to not have it taking up screen real estate. What are your thoughts, @jfrankl ?